src/main/java/Parser/Parser.java

```diff
@@ -0,0 +1,72 @@
+package Parser;
+
+import Commands.*;
+import Exceptions.InvalidCommandException;
+import Exceptions.InvalidDescriptionException;
+import Tasks.Deadline;
+import Tasks.Event;
+import Tasks.Todo;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Parser {
+
+    /**
+     * Interprets the user's input and returns the respective command.
+     * @param command user's input string
+     * @return command according to user's input
+     * @throws InvalidDescriptionException empty description of task
+     * @throws InvalidCommandException unknown command
+     */
+    public static Command parse(String command) throws InvalidDescriptionException, InvalidCommandException {
```

Comment:
You should consider catching exceptions as early as possible, throwing your exception outwards instead of handling it when you can will affect your program flow.